extensions/jira/src/helpers/withJiraCredentials.tsx

```diff
@@ -82,6 +141,12 @@ export function withJiraCredentials(component: JSX.Element) {
   return React.cloneElement(component, { query: query });
 }
 
+// TODO: Make this accessible from the UI
+// Allow a user to reselect a site
+export function reSelectSite() {
```

Comment:
This is currently unused/dead code (but linter isn't catching it because it's an export).

I want to shove this in the extension configuration screen, like where it allows you to `Logout` of Jira...but I haven't figured out yet where that code lives...